### PR TITLE
chore: extract version from a standalone file

### PR DESCRIPTION
**Pull request by Megrax**
FYI: https://github.com/bytebase/bytebase.com/pull/316#discussion_r915562227

### CODE REVIEW

File: VERSION.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
[the remark this answers is not in the document]
Comment (Megrax) [reply VERSION.json]: PTAL

File: tsconfig.json [hunk 2]
```diff
@@ -5,6 +5,7 @@
     "moduleResolution": "Node",
     "lib": ["ESNext", "ESNext.AsyncIterable", "DOM", "DOM.Iterable"],
     "esModuleInterop": true,
+    "resolveJsonModule": true,
```
Comment (tianzhou) [inline tsconfig.json:8]: do we still need this?

File: package.json [hunk 3]
```diff
@@ -65,6 +65,7 @@
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-vue": "^8.1.1",
     "postcss": "^8.3.5",
+    "raw-loader": "^4.0.2",
```
Comment (tianzhou) [inline package.json:68]: Didn't realize it requires an extra package to do this. Frontend is hard.

File: VERSION [hunk 4]
```diff
@@ -0,0 +1 @@
+1.2.0
```
Comment (tianzhou) [inline VERSION:1]: It should be 1.2.1 now

**Review by tianzhou — CHANGES_REQUESTED** [review]

**Pull request merged**